docs/ref/migration-operations.txt

```diff
@@ -345,6 +345,12 @@ set ``atomic=False``.
     ``schema_editor.connection.alias``, where ``schema_editor`` is the second
     argument to your function).
 
+.. warning::
+
+    When migrating data across two apps, ``dependencies`` should include the target
```

Comment:
I'd even go with "should include the latest migration it relies on of the target app".